### ISSUE TITLE
should .vscode folder be in the repo?

### ISSUE THREAD

**Issue by phinziegler** [phinziegler/SimBox#8]
My intuition tells me no, but I don't fully understand the syntax currently being used in .gitignore

**Comment by mirahunt** [comment]
No, it probably should not be because different developers have different preferences, and probably don't want to share each others' settings
